.circleci/config.yml

```
@@ -18,7 +18,7 @@ jobs:
           name: Install package dependencies
           command: |
             echo -e 'options(Ncpus = 6, repos = structure(c(CRAN = "https://cloud.r-project.org/")))' > $HOME/.Rprofile
-            R -q -e 'if (!requireNamespace("remotes")) install.packages("remotes")'
+            R -q -e 'if (!requireNamespace("remotes") || "remotes" %in% old.packages()) install.packages("remotes")'
```

`old.packages()` is slow, we only need to check {remotes}.

```suggestion
            R -q -e 'if (!requireNamespace("remotes") || packageVersion("remotes") < "x.y.z") ...'
```

---

We need to make this dynamic on the latest release version of _remotes_.
Is there a better way than the following overkill?
I don't want to have this in the YAML.

``` r
desc = readLines(url("https://raw.githubusercontent.com/r-lib/remotes/master/DESCRIPTION")) 
desc2 = desc[grep("Version:", desc)]
desc3 = strsplit(desc2, " ")[[1]][2]
substr(desc3, 1, 5)
#> [1] "2.1.0"
```

<sup>Created on 2019-12-01 by the [reprex package](https://reprex.tidyverse.org) (v0.3.0)</sup>

---

What about `install.packagges("remotes")` ? Sound like the least of all evils.

---

Costs us some seconds but yeah, maybe this is the simplest way to ensure max safety in this issue.